tests/test_e2ee_script_help.py

```
@@ -0,0 +1,62 @@
+SCRIPT_PATH = REPO_ROOT / "scripts" / "test_e2ee_integration.py"
+
+if not SCRIPT_PATH.exists():
+    pytest.skip(
+        f"Integration script not found: {SCRIPT_PATH}",
+        allow_module_level=True,
+    )
+
+
+@pytest.fixture(scope="module")
+def help_result():
+    return subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)